chart/templates/dag-processor/dag-processor-deployment.yaml

```diff
@@ -175,6 +175,35 @@ spec:
         {{- if and (.Values.dags.gitSync.enabled) (not .Values.dags.persistence.enabled) }}
           {{- include "git_sync_container" . | indent 8 }}
         {{- end }}
+        {{- if .Values.dagProcessor.logGroomerSidecar.enabled }}
+        - name: dag-processor-log-groomer
+          resources: {{- toYaml .Values.dagProcessor.logGroomerSidecar.resources | nindent 12 }}
+          image: {{ template "airflow_image" . }}
+          imagePullPolicy: {{ .Values.images.airflow.pullPolicy }}
+          {{- if .Values.dagProcessor.logGroomerSidecar.command }}
+          command: {{ tpl (toYaml .Values.dagProcessor.logGroomerSidecar.command) . | nindent 12 }}
+          {{- end }}
+          {{- if .Values.dagProcessor.logGroomerSidecar.args }}
+          args: {{- tpl (toYaml .Values.dagProcessor.logGroomerSidecar.args) . | nindent 12 }}
+          {{- end }}
+          {{- if .Values.dagProcessor.logGroomerSidecar.retentionDays }}
+          env:
+            - name: AIRFLOW__LOG_RETENTION_DAYS
+              value: "{{ .Values.dagProcessor.logGroomerSidecar.retentionDays }}"
+          {{- end }}
+          volumeMounts:
+            - name: logs
+              mountPath: {{ template "airflow_logs" . }}
+            {{- if .Values.volumeMounts }}
+              {{- toYaml .Values.volumeMounts | nindent 12 }}
+            {{- end }}
+            {{- if .Values.dagProcessor.extraVolumeMounts }}
+              {{- tpl (toYaml .Values.dagProcessor.extraVolumeMounts) . | nindent 12 }}
+            {{- end }}
+            {{- if or .Values.webserver.webserverConfig .Values.webserver.webserverConfigConfigMapName }}
```

Comment:
I guess having `webserver-config` in the list of `volumeMounts` requires declaration in `volumes` as well (missing in this PR). Here is an example from [triggerer](https://github.com/apache/airflow/blob/257136786c9a3eebbae717738637ab24fd6ab563/chart/templates/triggerer/triggerer-deployment.yaml#L248). Without it for me Helm chart 1.10.0 is not working.